src/server/vfs/mac/litesyncextconnector.mm

```diff
@@ -1405,20 +1440,23 @@ bool checkIoErrorAndLogIfNeeded(IoError ioError, const std::string &itemType, co
             continue;
         }
 
-        bool isPlaceholder;
-        bool isHydrated;
-        bool isSyncing;
-        int progress;
-        if (!vfsGetStatus(tmpPath, isPlaceholder, isHydrated, isSyncing, progress)) {
+        bool isChildPlaceholder;
+        bool isChildHydrated;
+        bool isChildSyncing;
+        int childProgress;
+        if (!vfsGetStatus(tmpPath, isChildPlaceholder, isChildHydrated, isChildSyncing, childProgress)) {
             continue;
         }
 
-        if (isSyncing) {
+        if (isChildSyncing) {
             existsOneStatusSync = true;
             break;
         }
 
-        existsOneStatusOnLine |= !isHydrated;
```

Comment:
You changed the logic here. Is it intended?

Reply:
YEs, if there is at least 1 dehydrated child, the parent must appear with the cloud icon.
The variables has been renamed though in order to improve readability.